### DISCUSSION TITLE
关于名字

### DISCUSSION THREAD

**Discussion by JiuXiat** [share121/inter-knot#4776] [其他]
JiuXiat，这个名字怎么样？
中文：九霄汀
大家猜猜是哪几个角色的名字？

**Comment by JiuXiat** [comment]
额，怎么改名字？

**Reply by ykrsama** [reply]
[更改 GitHub 用户名](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/changing-your-github-username#changing-your-username)

**Reply by JiuXiat** [reply]
谢了

**Comment by Robin-txdy** [comment]
第一个不知道
第二个飞霄？
第三个维尔汀？

**Reply by JiuXiat** [reply]
虽然没有完全猜对，但还是要鼓励你的，猜对了一个
正确答案：蓬莱寺九霄，维尔汀

**Reply by Robin-txdy** [reply]
至少是猜对了一个<img src="https://bbs-static.miyoushe.com/static/2023/05/22/a2fa805a0a782cdb8ab44ddaf8d5343d_9213146029750742963.png" alt="阿姬倒地了02" width="75" >

**Reply by 596wjr** [reply]
> 虽然没有完全猜对，但还是要鼓励你的，猜对了一个 正确答案：蓬莱寺九霄，维尔汀

崩崩？

**Reply by JiuXiat** [reply]
是的
